### DISCUSSION TITLE
We have our own trade history with our own strategy. Can we use this library?

### DISCUSSION THREAD

**Discussion by JamesDuncanRobertson** [kernc/backtesting.py#702] [Q&A]
I am working with a team. We have OHLC data and alongside it, we have predictions UP & DOWN. 

Can we use this library to get all these interesting backtest metrics that this library is capable of (like Drawdown, Equity balance, Sharpe Ratio, vs Buy&Hold, etc). We would love to see some of these. 

We don't want to use any of the library's strategies, but keep our own in the data. Can this be done?

Thanks, and sorry if my question is really basic.

**Comment by kernc** [comment]
Sure, see: https://github.com/kernc/backtesting.py/issues/64.
